raiden/tests/fixtures/variables.py

```diff
@@ -52,6 +53,27 @@ def events_poll_timeout():
 
 
 @pytest.fixture
+def random_marker():
+    """ A random marker used to identify a pytest run.
+
+    Some tests will spawn a private chain, the private chain will be one or
+    more ethereum nodes on a new subprocesss. These nodes may fail to start on
+    concurrent test runs, mostly because of port number conflicts, but even
+    though the test fails to start it's private chain it may run interacting
+    with the geth process from the different test run! This leads to
+    unreasonable test errors.
+
+    This fixture creates a random marker used to distinguish pytest runs and
+    avoid test failures. Note this is not fail blinded and may fail to detect
```

Comment:
fail blinded?

Reply:
Yeah, I couldn't think of any word to convey the meaning that the test may fail for other reasons

Reply:
Oh this is what you wanted to say. So perhaps rephrase:

"Note this could fail for other reasons and fail to detect..."  
?

Reply:
Yeah, sounds better,thanks